docs/DeveloperGuide.md

```diff
@@ -502,148 +502,174 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
-    Use case ends.
+**MSS** <br>
+1. User requests to filter the list of companies by one of the application statuses. 
+2. LinkMein displays a list of companies matching the application status. <br>
+Use case ends.
 
-**Use case: View full company information**
+**Extensions** <br>
+* 1a. LinkMeIn detects an invalid command format error in the input →  handled similarly to 1a of UC01. 
+* 1b. LinkMeIn detects an invalid parameter input → handled similarly to 1b of UC02.
 
-**MSS**
 
-1. User requests to list companies.
-2. AddressBook shows a list of companies.
-3. User requests to view a specific company in the list.
-4. AddressBook shows the full information of the company in the company detail panel.
+**Use Case: UC09 - Sort Companies by Deadline**
 
-    Use case ends.
+**MSS** <br>
+1. User requests to sort the list of companies by deadline. 
+2. LinkMeIn displays the sorted list of companies. <br>
+Use case ends.
 
-**Extensions**
+**Extensions** <br>
+* 1a. LinkMeIn detects an invalid command format error in the input →  handled similarly to 1a of UC01. 
+* 1b. LinkMeIn detects an invalid parameter input → handled similarly to 1b of UC02.
 
--   2a. The list is empty.
 
-    Use case ends.
+**Use Case: UC10 - Add Remarks for a Company**
 
--   3a. The given index is invalid.
+**MSS** <br>
+1. User requests to add remarks for a specific company. 
+2. LinkMeIn adds the remarks to the company. <br>
+Use case ends.
 
-    -   3a1. AddressBook shows an error message.
+**Extensions** <br>
+* 1a. LinkMeIn detects an invalid index input → handled similarly to 1a of UC03.
 
-        Use case resumes at step 2.
 
+**Use Case: UC11 - Delete Remarks for a Company**
 
-**Use case: Sort companies by deadline**
+**MSS** <br>
+1. User requests to delete remarks for a specific company from the list of companies. 
+2. LinkMeIn deletes the remarks from the company. <br>
+Use case ends.
 
-**MSS**
+**Extensions** <br>
+* 1a. LinkMeIn detects an invalid index input → handled similarly to 1a of UC03.
 
-1.  User requests to list companies.
 
-2.  AddressBook shows a list of companies.
+**Use Case: UC12 - Exit the Program**
 
-3.  User requests to sort the companies by deadline in a specific order (ascending or descending).
-
-4.  AddressBook sorts and displays the companies based on the deadline in the specified order.
-
-    Use case ends.
-
-
-**Extensions**
+**MSS** <br>
+1. User requests to exit the program. 
+2. LinkMeIn exits the program. <br>
+Use case ends.
 
--   2a. The list is empty.
+**Extensions** <br>
+* 1a. LinkMeIn detects an invalid command format error in the input →  handled similarly to 1a of UC01.
 
-    Use case ends.
 
--   3a. The given order (ascending or descending) is invalid or not specified.
+**Use Case: UC13 - View Help**
 
-    -   3a1. AddressBook shows an error message.
+**MSS** <br>
+1. User requests for help to use LinkMeIn. 
+2. LinkMeIn displays a link to the user guide. <br>
```

Comment:
Should be "Displays a window with a link to the user guide"? Will that make more sense

Reply:
yup okay!